docs/apache-airflow/upgrading-to-2.rst

```diff
@@ -287,7 +287,7 @@ When DAGs are initialized with the ``access_control`` variable set, any usage of
     Breaking change
 
     Previously we were using two versions of UI, which were hard to maintain as we need to implement/update the same feature
-    in both versions. With this release we've removed the older UI in favor of Flask App Builder RBAC UI. No need to set the
+    in both versions. With this release we have removed the older UI in favor of Flask App Builder RBAC UI. No need to set the
```

Comment:
Some grammar nits, recommended rewrite here:

Previously we were using two versions of the UI. This was difficult to maintain, because it meant we had to implement/update features in two places. With this release, we have removed the older UI in favor of the Flask App Builder RBAC UI, reducing a huge maintenance burden. There is no longer a need to set the RBAC UI explicitly in the configuration, as it is the only default UI.

Reply:
I like it